source/routes/delete/daily.js

```diff
@@ -0,0 +1,37 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/* *
+Delete daily entry json object and all bullet documents in that daily entry.
+*/
+module.exports = {
+    '/delete/daily': {
+        methods: ['delete'],
+        fn: function(req, res, next) {
+            //get daily entry by id
+            db.get(req.body._id)
+            .then((response) => {
+                //access all bullet documents' id
+                response.bullets.forEach((bullet, index) => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 16.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7422202)
```suggestion
            response.bullets.forEach((bullet, index) => {
```